sources/bash_completion.d/swizzin

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# ln -fs /etc/swizzin/sources/bash_completion.d/swizzin /etc/bash_completion.d/swizzin
+#
+_command_complete() {
+    local cur prev
+    cur=${COMP_WORDS[COMP_CWORD]}
+    prev=${COMP_WORDS[COMP_CWORD - 1]}
+    readarray -d ' ' apps_array < <(find "/etc/swizzin/scripts/install/" -maxdepth 1 -mindepth 1 -type f -printf "%f ")
```

Comment:
What if we remove things that are already installed from this, i.e. remove from this array based on what's in in `/install`? Is it worth the trouble?